ansible/provision.yaml

```diff
@@ -1,4 +1,13 @@
 ---
+- hosts: all
+  name: Validate Variables
+  gather_facts: false
+  tasks:
+    - name: Ensure `pause_image` is set when `download_images` is `false`
+      fail:
+        msg: Variable `pause_image` must be set if `download_images is `false`
+      when: not download_images
```

Comment:
Is this still needed, this var is being set in https://github.com/mesosphere/konvoy-image-builder/pull/161/files#diff-14c84fc26cc68826bdd8c9ee2ba0b29f31537fc6fec4f03028d38b024d7b9d6cR31-R37?

Reply:
It was, since that was only being set to the correct value _after_ the `images` step, and that step only runs when `download_images` was true and we needed the right value for the first `config_containerd`. Now however, it will always be set by the time `config` (since the `version`) role is running before it, and we no longer have to check it to be set for "offline" installs.